package_manifest.yaml

```
@@ -12,6 +12,9 @@ packages:
   vars:
     rpmlint_rc_file: "{{ inventory_dir }}/.rpmlintrc"
     build_package_use_koji_build: true
+    macros:
+      foremandist: "{{ foremandist }}"
+      nightly: "{{ '.' + lookup('pipe', 'date +%Y%m%d%H%M%S') + 'git' + nightly_githash[:7] if nightly_githash is defined else omit }}"
```

I had to try it out to realize the `else omit` actually wipes *the whole thing* when `nightly_githash` is not set.
```suggestion
      nightly: "{{ ('.' + lookup('pipe', 'date +%Y%m%d%H%M%S') + 'git' + nightly_githash[:7]) if nightly_githash is defined else omit }}"
```
IMHO this makes it more readable?